### PR TITLE
DX: use method expectDeprecation from Symfony Bridge instead of annotation

### PR DESCRIPTION
**Pull request by kubawerlos**

### CODE REVIEW

File: tests/Console/Command/FixCommandTest.php [hunk 1]
```diff
@@ -26,6 +27,8 @@
  */
 final class FixCommandTest extends TestCase
 {
+    use ExpectDeprecationTrait;
```
Comment (keradus) [inline tests/Console/Command/FixCommandTest.php:30]: let's import it directly to `TestCase`

File: .php_cs.dist [hunk 2]
```diff
@@ -27,6 +27,7 @@ $config
         '@PHPUnit75Migration:risky' => true,
         '@PhpCsFixer' => true,
         '@PhpCsFixer:risky' => true,
+        'general_phpdoc_annotation_remove' => ['annotations' => ['expectedDeprecation']],
```
Comment (keradus) [inline .php_cs.dist:30]: can you create a feature request to autoconvert annotation to method call?

Comment (kubawerlos) [reply .php_cs.dist:30]: done: https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/5616

**Review by GrahamCampbell — APPROVED** [review]

**Comment by keradus** [comment]
Thank you @kubawerlos.

**Pull request merged**